ArduPlane/quadplane.cpp

```diff
@@ -843,7 +843,7 @@ void QuadPlane::run_esc_calibration(void)
  */
 void QuadPlane::multicopter_attitude_rate_update(float yaw_rate_cds)
 {
-    bool use_multicopter_control = in_vtol_mode() && !tailsitter.in_vtol_transition();
+    bool use_multicopter_control = !use_fw_attitude_controllers();
```

Comment:
I cant take off the H wing with this change.

Reply:
i've tested both QLOITER takeoff and AUTO takeoff in the RF9 HWing model with this branch with no issue. Can you give me a log showing a failure?

Reply:
@tridge try in Qstab. Qloiter and AUTO don't use this code path. `use_fw_attitude_controllers()` has a `!tailsitter.enabled()` in it so always returns true, hence we never use multi rotor control.

We could puzzle out the exact combination to make it work, but I don't think it should be in this pr. Tailsitters also want to run both controllers sometimes, see the motors only Qassist option.